src/Tutorials/Basics/hda_reaction.py

```diff
@@ -20,11 +20,9 @@
 # Import Pyomo libraries
 from pyomo.environ import (Constraint,
                            exp,
-                           Expression,
-                           Param,
-                           PositiveReals,
                            Set,
                            Var,
+                           Param,
```

Comment:
Just to clarify, why do we have two versions of the HDA reactions in the examples? What is different between the two?

If they are the same, then maybe we need a separate `common` folder (maybe a better name) to hold common resources like this.

Reply:
The two copies of the thermodynamic properties and reaction properties are just for imports, and I agree that having a common folder (e.g. `src/commonfiles`) for shared packages, modules, CSV files, etc would help clean up the examples repository.

Reply:
If you prefer, I could add a commit to add the common folder in this PR, and then we open another PR later to move other files.